shapepipe/pipeline/file_handler.py

```diff
@@ -1120,6 +1135,9 @@ def _format_process_list(
             ])
             process_list.append(process_items)
 
+        if len(process_list) == 0:
```

Comment:
Catching an error that sometimes occurs, e.g. when something is wrong with input files or directories.
If not, caused unexpected behaviour later on. Check and error message could still be improved.